hy/core/language.hy

```diff
@@ -226,14 +226,16 @@
        => (disassemble '(print \"Hello World!\") True)
        print('Hello World!')
   "
-  (import astor)
-  (import hy.compiler)
+  (import ast hy.compiler hy._compat)
 
   (setv compiled (hy.compiler.hy-compile tree (calling-module-name) :import-stdlib False))
-  ((if codegen
-       astor.code-gen.to-source
-       astor.dump-tree)
-    compiled))
+  (if
+    codegen
+      (hy._compat.ast-unparse compiled)
+    hy._compat.PY3_9
+      (ast.dump compiled :indent 1)
+    True
+      (ast.dump compiled)))
```

Comment:
Isn't the last form of an `if` already implicitly an `else`?

Reply:
Yes, but I think it's easier to read with the placeholder `True`.